app/stac_api/views.py

```diff
@@ -527,7 +529,8 @@ def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
 
 
-class TestHttp500(AssetDetail):
+class TestHttp500(generics.GenericAPIView):
+    queryset = LandingPage.objects.all()
```

Comment:
:+1:
Guess this stops the `CRITICAL` in the log when running tests

Reply:
Not really it always have a CRITICAL message because that what we want to test, but the message would be about the `AttributeError('test exception')`

Reply:
OK, but probably it sounds less "panicy" now :wink: